_build/templates/default/sass/login.scss

```diff
@@ -183,3 +180,46 @@ a {
   border: 1px solid #aaa;
   position: relative;
 }
+
+@media (max-width: 575.98px) {
```

Comment:
For consistency reasons I suggest to keep the max-width equal to the one set for the mobile responsive breakpoint: https://github.com/modxcms/revolution/blob/2.x/_build/templates/default/sass/_colors-and-vars.scss#L193-L204